graphql_compiler/schema/__init__.py

```diff
@@ -242,8 +242,8 @@ def _parse_datetime_value(value):
         # Strip the "Z" and add an explicit time zone instead.
         value = value[:-1] + '+00:00'
 
-    return arrow.get(value, 'YYYY-MM-DDTHH:mm:ssZ').datetime
 
+    return arrow.get(value, 'YYYY-MM-DDTHH:mm:ssZZ').datetime
```

Comment:
If this is conditional upon a specific arrow version to keep working, then let's set that version as a minimum in our `setup.py`.

Reply:
This just seems to have been the right way always. 

I ran the datetime parsing tests using this format with arrow versions 0.7.0, (the earliest arrow version specified allowed by setup.py) and 0.15.0, (the current latest version). 

On a separate note, the package requirements in the Pipfile file seem to be out of sync with the setup.py file. Do you know why that may be?